plugins/woocommerce/tests/api-core-tests/tests/customers/customers-crud.test.js

```diff
@@ -35,18 +51,22 @@ test.describe('Customers API tests: CRUD', () => {
 			expect(response.status()).toEqual(200);
 			expect(responseJSON.is_paying_customer).toEqual(false);
 			expect(responseJSON.role).toEqual('administrator');
-			expect(responseJSON.username).toEqual('admin');
+			// this test was updated to allow for local test setup and JN
+			expect(['admin', 'demo']).toContain(responseJSON.username);
```

Comment:
Perhaps we can make this more configurable--where the user can setup an admin username of choice, and then we aren't tied to only being able to use a site with `admin` or a site with `demo`. So maybe we can add en an env value of something like `ADMIN_USER` that we can use here.